examples/credential-registry/Cargo.toml

```diff
@@ -6,7 +6,7 @@ version = "0.1.0"
 edition = "2021"
 license = "MPL-2.0"
 authors = [ "Concordium <developers@concordium.com>" ]
-description = "An example credential registry smart contract"
+description = "Example-credential-registry"
```

Comment:
Why the dashes? Description should be free-form text typically.

Reply:
It seems like GitHub CI cannot parse strings with spaces properly. Enclosing it in (escaped) quotes of various kinds didn't help. I asked @DOBEN, and it seems like the solution is to use dashes or underscores at the moment. It's annoying indeed that because of the CI, I had to change the description (I should be able to pass the same description in the CI as in the example when comparing the generated contract). The same happened with the NFT contract.

Reply:
I missed where you need to parse this string.

Reply:
Here is why the build fails: https://github.com/Concordium/concordium-rust-smart-contracts/actions/runs/6011591168/job/16305264615#step:4:12
It shows how the description is parsed (I guess it's lexing, not parsing though)

Reply:
Tried multiple options to escape spaces and quotes - non of them worked.